src/Common/src/Interop/Windows/mincore/Interop.GetConsoleTitle.cs

```diff
@@ -0,0 +1,61 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Diagnostics;
+using System.Text;
+using System.Runtime.InteropServices;
+
+internal partial class Interop
+{
+    internal partial class mincore
+    {
+        [DllImport(Libraries.Console_L1, CharSet = CharSet.Unicode, SetLastError = true, EntryPoint = "GetConsoleTitleW")]
+        private static extern int GetConsoleTitle(StringBuilder title, int nSize);
```

Comment:
Console.Title implementation is not a direct port, since on Desktop it used JitHelper APIs that the portable library can't depend on but it is pretty similar :). Also, I do think StringBuilder better represents buffers of variable lengths. Adding [Out] though for explicit annotation. :)